dapps/react-dapp-auth/components/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,59 @@
+import { Flex, Switch } from "@chakra-ui/react";
+import { useCallback, useEffect, useMemo, useState } from "react";
+
+const ThemeSwitcher: React.FC = () => {
+  const [darkMode, setDarkMode] = useState(true);
+
+  const darkTheme = useMemo(
```

Comment:
Not worth refactoring right now, but just an FYI that Chakra also has functionality like `extendTheme` which can be used directly with `ChakraProvider`:
https://chakra-ui.com/docs/styled-system/customize-theme

Reply:
Forgot about that mb